lib/mix/tasks/phx.gen.socket.ex

```diff
@@ -0,0 +1,94 @@
+defmodule Mix.Tasks.Phx.Gen.Socket do
+  @shortdoc "Generates a Phoenix socket handler"
+
+  @moduledoc """
+  Generates a Phoenix socket handler.
+
+      mix phx.gen.socket User
+
+  Accepts the module name for the socket
+
+  The generated files will contain:
+
+  For a regular application:
+
+    * a socket in `lib/my_app_web/channels`
+    * a socket test in `test/my_app_web/channels`
+
+  For an umbrella application:
+
+    * a socket in `apps/my_app_web/lib/app_name_web/channels`
+    * a socket test in `apps/my_app_web/test/my_app_web/channels`
+
```

Comment:
```suggestion
  For a regular application:

    * a client in `assets/js`
    * a socket in `lib/my_app_web/channels`
    * a socket test in `test/my_app_web/channels`

  For an umbrella application:

    * a client in `apps/my_app_web/assets/js`
    * a socket in `apps/my_app_web/lib/app_name_web/channels`
    * a socket test in `apps/my_app_web/test/my_app_web/channels`

```

Reply:
I changed in the commit faaf240e189e9b2622640460d1f067da597ca256